plugin/http/Response.lua

```diff
@@ -28,7 +28,12 @@ function Response:isSuccess()
 end
 
 function Response:json()
-	return HttpService:JSONDecode(self.body)
+	return Encode.jsonDecode(self.body)
 end
 
+function Response.setEncode(encode)
```

Comment:
Inject the encode functions here to avoid a circular dependency between Response and Http modules